payload-schemas/schemas/common/repository.schema.json

```diff
@@ -429,6 +429,9 @@
         }
       },
       "additionalProperties": false
+    },
+    "public": {
+      "type": "boolean"
```

Comment:
I'm pretty sure that the `"public"` key is always present, can you add it to the `"required"` array?

Reply:
The `"private"` key is always present, but not `"public"`.
I checked the payload for `repository.transferred`, `repository.created` and the `"public"` key is not included in the `"repository"` object.

It seems that it's only used for the `fork` event from what I can tell

Reply:
You are right, sorry, I mixed up the two